include/seqan/find/find_base.h

```diff
@@ -249,13 +249,13 @@ struct Needle<Segment<THost, TSpec> const>
  * Finder<CharString> finder(hstck);
  *
  * Pattern<CharString, Horspool> p1("y");
- * findAll(finder, p1);
+ * find(finder, p1);
```

Comment:
Thanks for spotting this. Hard coded code snippets are anyway evil.
Please use `@include demos/dox/find/finder_online.cpp` to include a compiled and executed code snippet.
You can also add `@include demos/dox/find/finder_online.cpp.stdout` to view the result of this example.

Like so: 
```
 /*!
 * ...
 * @include demos/dox/find/finder_online.cpp
 * 
 * The result is as follows
 * 
 * @include demos/dox/find/finder_online.cpp.stdout
 * /
```